reladomo/src/main/java/com/gs/fw/finder/attribute/TimestampAttribute.java

```diff
@@ -1,43 +1,43 @@
-
-/*
- Copyright 2016 Goldman Sachs.
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing,
- software distributed under the License is distributed on an
- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- KIND, either express or implied.  See the License for the
- specific language governing permissions and limitations
- under the License.
- */
-
-package com.gs.fw.finder.attribute;
-
-import com.gs.fw.finder.Attribute;
-import com.gs.fw.finder.Operation;
-
-import java.sql.Timestamp;
-import java.util.Set;
-
-public interface TimestampAttribute<Owner> extends Attribute<Owner>
-{
-    public Operation<Owner> eq(Timestamp value);
-
-    public Operation<Owner> notEq(Timestamp value);
-
-    public Operation<Owner> greaterThan(Timestamp value);
-
-    public Operation<Owner> greaterThanEquals(Timestamp value);
-
-    public Operation<Owner> lessThan(Timestamp value);
-
-    public Operation<Owner> lessThanEquals(Timestamp value);
-
-    public Operation<Owner> in(Set<Timestamp> timestampSet);
-
-    public Operation<Owner> notIn(Set<Timestamp> timestampSet);
-}
+
+/*
+ Copyright 2016 Goldman Sachs.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ */
+
+package com.gs.fw.finder.attribute;
+
+import com.gs.fw.finder.Attribute;
+import com.gs.fw.finder.Operation;
+
+import java.sql.Timestamp;
+import java.util.Set;
+
+public interface TimestampAttribute<Owner> extends Attribute<Owner>
+{
+    Operation<Owner> eq(Timestamp value);
```

Comment:
let's not mix this with more edits (it's big enough as it is)

Reply:
will do.
 BTW, you mentioned that you have a draft of the JDK8 migration. Please share the branch.